extension/src/services/binding.ts

```diff
@@ -331,7 +368,7 @@ export default class Binding {
 
             chrome.runtime.sendMessage(command);
 
-            if (this._synced) {
+            if (this._synced && this._playMode !== PlayMode.fastForward) {
```

Comment:
I think the constant speed notification is annoying so I've disabled it for the new mode